beacon_node/beacon_chain/src/data_availability_checker/overflow_lru_cache.rs

```diff
@@ -230,15 +230,48 @@ impl<E: EthSpec> PendingComponents<E> {
         self.merge_blobs(reinsert);
     }
 
-    /// Checks if the block and all of its expected blobs are available in the cache.
+    /// Checks if the block and all of its expected blobs or custody columns (post-PeerDAS) are
+    /// available in the cache.
     ///
-    /// Returns `true` if both the block exists and the number of received blobs matches the number
-    /// of expected blobs.
-    pub fn is_available(&self) -> bool {
-        if let Some(num_expected_blobs) = self.num_expected_blobs() {
-            num_expected_blobs == self.num_received_blobs()
-        } else {
-            false
+    /// Returns `true` if both the block exists and the number of received blobs / custody columns
+    /// matches the number of expected blobs / custody columns.
+    pub fn is_available(
+        &self,
+        block_import_requirement: BlockImportRequirement,
+        log: &Logger,
+    ) -> bool {
+        match block_import_requirement {
+            BlockImportRequirement::AllBlobs => {
+                debug!(
+                    log,
+                    "Checking block and blob importability";
+                    "block_root" => %self.block_root,
+                    "num_expected_blobs" => ?self.num_expected_blobs(),
+                    "num_received_blobs" => self.num_received_blobs(),
+                );
+                if let Some(num_expected_blobs) = self.num_expected_blobs() {
+                    num_expected_blobs == self.num_received_blobs()
+                } else {
+                    false
+                }
+            }
+            BlockImportRequirement::CustodyColumns(num_expected_columns) => {
+                let num_received_data_columns = self.num_received_data_columns();
+
+                debug!(
+                    log,
+                    "Checking block and data column importability";
+                    "block_root" => %self.block_root,
+                    "num_received_data_columns" => num_received_data_columns,
+                );
```

Comment:
Yep agree that it could be become noisy, and I thought about downgrading it too - it was quite useful when I was debugging issues with block importability. I guess we could get this somehow with other logs though. Will downgrade to `trace`. (Also not opposed to just removing it)

Reply:
Addressed in [3424dcd](https://github.com/sigp/lighthouse/pull/5570/commits/3424dcd36ee14c3ebcdf21469a83e5a698677ef3)